src/slack/message/index.js

```diff
@@ -11,12 +17,25 @@ const getSection = (text) => ({
   },
 });
 
+/**
+ * Get text for reminders.
```

Comment:
A better description might be: Get text for reminders or new errand

Reply:
Yep, makes sense.